### PR TITLE
Redirect to /login on logout instead of /urls

### PR DESCRIPTION
**Pull request by Michael-Xie**

### CODE REVIEW

File: routes/sessions/session.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ module.exports = () => {
   // POST /logout - Log user out by removing encrypted cookie
```
Comment (khizer-ifiveo) [inline routes/sessions/session.js:69]: I don't understand why this change is necessary. Can you please explain!

File: routes/sessions/session.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ module.exports = () => {
...
   router.post("/logout", (req, res) => {
     req.session = null;
-    res.redirect("/urls");
+    res.redirect("/login");
   });
   return router;
```
Comment (khizer-ifiveo) [inline routes/sessions/session.js:74]: What does this mean?

**Review by khizer-ifiveo — COMMENTED** [review]
**Summary of Comments**

File: routes/sessions/session.js (hunk 1 above)
Comment (Michael-Xie) [reply routes/sessions/session.js:69]: sure, this is blah blah

File: routes/sessions/session.js (hunk 2 above)
Comment (Michael-Xie) [reply routes/sessions/session.js:74]: djfdkkfd

**Review by Michael-Xie — COMMENTED** [review]
feed back to comments

**Review by khizer-ifiveo — APPROVED** [review]
Looks good to me now.
